resources/scripts/components/server/files/FileManagerContainer.tsx

```diff
@@ -72,11 +89,14 @@ export default () => {
                         }
                     />
                 </ErrorBoundary>
+
+                <input onChange={searchFiles} css={tw`w-full rounded-lg bg-neutral-700 border-2 border-cyan-700 mx-10 px-4`} placeholder='search' style={{borderColor: 'rgb(59 130 246);'}}></input>
+
                 <Can action={'file.create'}>
                     <ErrorBoundary>
                         <div css={tw`flex flex-shrink-0 flex-wrap-reverse md:flex-nowrap justify-end mb-4 md:mb-0 ml-0 md:ml-auto`}>
-                            <NewDirectoryButton css={tw`w-full flex-none mt-4 sm:mt-0 sm:w-auto sm:mr-4`}/>
-                            <UploadButton css={tw`flex-1 mr-4 sm:flex-none sm:mt-0`}/>
+                            <NewDirectoryButton css={tw`w-full flex-none mt-4 sm:mt-0 sm:w-auto sm:mr-4`} />
+                            <UploadButton css={tw`flex-1 mr-4 sm:flex-none sm:mt-0`} />
```

Comment:
Remove space

Reply:
Fixed in https://github.com/pterodactyl/panel/pull/3926/commits/9862de5fd97dade3f9783d4d07a451504e830143